dependencies.gradle

```diff
@@ -1,5 +1,5 @@
 dependencies {
-    api('com.github.GTNewHorizons:GT5-Unofficial:5.09.44.104:dev')
+    api('com.github.GTNewHorizons:GT5-Unofficial:5.09.44.105-pre:dev')
```

Comment:
Aren't we supposed to not depend on -pre versions due to periodic purging?

Reply:
i fix that one sec @Connor-Colenso just spprove it i make a gt tag